microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -22,89 +22,102 @@ ms.date: 05/04/2021
    limitations under the License.  -->
 # View page resources
 
-Resources are the files that a page needs in order to display correctly.  Examples of resources include:
+Resources are the files that a page needs to display correctly.  Examples of resources include:
 *  CSS files.
 *  JavaScript files.
 *  HTML files.
 *  Image files.
 
 You can view the resources that make up a web page from within several tools, or panels.
 
-This guide assumes that you are familiar with the basics of [web development](https://developer.mozilla.org/docs/Learn) and [Microsoft Edge DevTools](../../devtools-guide-chromium/overview.md).
-
+This guide assumes that you are familiar with the basics of web development described in [Learn web development](https://developer.mozilla.org/docs/Learn) and [Microsoft Edge DevTools](../../devtools-guide-chromium/overview.md).
```

Comment:
```suggestion
For web development basics, see [Learn web development](https://developer.mozilla.org/docs/Learn).  See also [Overview of DevTools](../../devtools-guide-chromium/overview.md).
```
Re-frame.  Article makes no assumptions & requirements re: previous knowledge.  Offer "If you want... link".  Maybe copy links to See Also at bottom of article.